executors/node/collator_nonignorable.js

```diff
@@ -0,0 +1,44 @@
+// The Collator for nonignorable short tests.
+
+// !!! TODO: Collation: determine the sensitivity that corresponds
+// to the strength.
+module.exports = {
+
+  testCollationNonignorableShort: function(json) {
+
+    // Global default locale
+    let testLocale = '';
+    let testCollOptions = {};
+
+    // Set up collator object with optional locale and testOptions.
+    let coll;
+    try {
+      if (testLocale) {
+        coll = new Intl.Collator(testLocale, testCollOptions);
+      } else {
+        coll = new Intl.Collator(testCollOptions);
+      }
```

Comment:
The collator needs to be created with locale `undefined`; it does not automatically detect that the first argument is not a locale